healdata.org/manifests/hatchery/hatchery.json

```diff
@@ -134,13 +247,14 @@
       "name": "(Tutorial) JCOIN Tracking Opioid Stigma Notebook",
       "image": "quay.io/cdis/jupyter-jcoin:master",
       "env": {
-        "FRAME_ANCESTORS": "https://healdata.org"
+        "FRAME_ANCESTORS": "https://preprod.healdata.org"
```

Comment:
some urls are pointing to preprod

Reply:
Oof! Good catch, I copy pasted it from preprod to prod. I will fix it now!